src/browser/connection/index.ts

```diff
@@ -8,15 +10,22 @@ import nanoid from 'nanoid';
 import COMMAND from './command';
 import BrowserConnectionStatus from './status';
 import HeartbeatStatus from './heartbeat-status';
-import { GeneralError } from '../../errors/runtime';
+import { GeneralError, TimeoutError } from '../../errors/runtime';
 import { RUNTIME_ERRORS } from '../../errors/types';
-import { BROWSER_RESTART_TIMEOUT, HEARTBEAT_TIMEOUT } from '../../utils/browser-connection-timeouts';
 import { Dictionary } from '../../configuration/interfaces';
 import BrowserConnectionGateway from './gateway';
 import BrowserJob from '../../runner/browser-job';
 import WarningLog from '../../notifications/warning-log';
 import BrowserProvider from '../provider';
 
+import {
+    BROWSER_RESTART_TIMEOUT,
+    BROWSER_CLOSE_TIMEOUT,
+    HEARTBEAT_TIMEOUT
+} from '../../utils/browser-connection-timeouts';
+
+const DEBUG_SCOPE = (id: string): string => `testcafe:browser:connection:${id}`;
```

Comment:
I suggest to rename this constant as `const getBrowserConnectionDebugScope = (id: string)...`